### ISSUE TITLE
is Vue 3 supported?

### ISSUE THREAD

**Issue by cetricokola** [lusaxweb/vuesax-next#285]
Can i use vuesax in my vue 3 project?

**Comment by Xiaojiju** [comment]
following

**Comment by lk77** [comment]
This version of vuesax was meant to work with Vue 3, but i don't think it works now, it's not maintained,
I made a fork of original vuesax (aka https://github.com/lusaxweb/vuesax), that supports Vue 3 :

https://www.npmjs.com/package/vuesax3

**Comment by cetricokola** [comment]
Thank you. Let me check it out.

**Comment by Mikekkamara** [comment]
> This version of vuesax was meant to work with Vue 3, but i don't think it works now, it's not maintained, I made a fork of original vuesax (aka https://github.com/lusaxweb/vuesax), that supports Vue 3 :
> 
> https://www.npmjs.com/package/vuesax3

This was nice of you, but the ui components used here are for vuesax v3, not v4?

**Comment by lk77** [comment]
@Mikekkamara yes, original vuesax v3 https://github.com/lusaxweb/vuesax, not vuesax-next (v4)
